tools/data_processing_scripts/process_templated_for_gt.py

```diff
@@ -0,0 +1,49 @@
+"""
+This script scrapes player logs from S3 and grabs the commands that are most played.
```

Comment:
A nit: This is not what the script does :)
Let's rewrite the docstring ?

Reply:
Good catch! 

BTW, I'm going to be checking in more data processing scripts in `tools/data_processing_scripts` for reuse and to automate more aspects of data processing/model training.

Reply:
I had started scraping the logs..but quickly saw the bigger problem was duplicates. Keeping this in mind for future improvements :)